ui/package.json

```diff
@@ -6,20 +6,19 @@
     "build-terra-deps": "./src/terraui/build.sh",
     "deps": "yarn run codegen && yarn run build-terra-deps",
     "start": "BROWSER='none' PORT=4200 react-app-rewired start",
-    "start-tanagra": "npm install --prefix ../tanagra/ui && npm run codegen --prefix ../tanagra/ui && BROWSER='none' REACT_APP_POST_MESSAGE_ORIGIN=http://localhost:4200 npm start --prefix ../tanagra/ui",
+    "start-tanagra": "BROWSER='none' REACT_APP_POST_MESSAGE_ORIGIN=http://localhost:4200 npm start --prefix ../tanagra-aou-utils/tanagra/ui",
     "dev-up": "yarn && yarn run deps && yarn start",
     "dev-up-test": "yarn && yarn run deps && REACT_APP_ENVIRONMENT=localtest yarn start",
     "dev-up-local": "yarn && yarn run deps && REACT_APP_ENVIRONMENT=local yarn start",
-    "dev-up-tanagra-local": "yarn && concurrently \"yarn run start-tanagra\" \"yarn run dev-up-local\"",
+    "dev-up-tanagra-local": "./src/tanagra-ui/dev-up-tanagra-local.sh",
     "build": "react-app-rewired build",
     "test:debug": "react-app-rewired --inspect-brk test --watchAll=false --run-in-band",
     "test": "react-app-rewired test --watchAll=false",
     "lint": "yarn eslint src --ext .ts,.tsx",
-    "codegen": "yarn run codegen-clean && yarn run codegen-swagger && yarn run notebooks-codegen-swagger && yarn run tanagra-codegen-openapi",
```

Comment:
I think we need to keep the tanagra codegen here since we have imports referencing it in the workbench code.